docs/guide/database-access.md

```diff
@@ -0,0 +1,117 @@
+# Accessing the database directly
+
+## Working with collections
+As stated earlier, the main goal of the Apostrophe content query API is to facilitate requests to the MongoDB database without the developer having to know advanced MongoDB syntax or worry about user permissions. However, there are instances where you might want to directly access and modify your collections without the overhead or restrictions of the API.
+
+A good example is updating a "view counter" on a piece like a blog article. Everyone can see the page, so you don't care about permissions. You don't need to update the entire document object. You just want to update one small piece of it.
+
+Here is an example of how to do that in a piece. We will use the `beforeShow()` method of the `@apostrophecms/piece-page-type`. This method lets us make modifications just before the `show.html` of the associated `@apostrophe/piece-page-type` is displayed to the user.
+
+<AposCodeBlock>
+
+```javascript
+module.exports = {
+  // remainder of the module properties
+  methods(self) {
+    return {
+      beforeShow(req){
+        self.apos.doc.db.updateOne(
+          { _id: req.data.piece._id },
+          { $inc: { views: 1 } }
+        );
+      }
+    };
+  }
+};
+```
+
+<template v-slot:caption>
+modules/article-page/index.js
+</template>
+</AposCodeBlock>
+
+Within the `beforeShow()` method we are gaining access to the `aposDocs` collection of the project database using `self.apos.doc.db`. This reference to the collection is shorthand for `self.apos.db.collection('aposDocs')`. Note that this is *not* the same thing as `self.apos.db`, which is the main MongoDB connection object. 
+
+::: tip
+There are references to other core apostrophe collections that can be accessed in this same way through `self.apos.<xxx>.db`. For example, `self.apos.attachment.db` gives access to CRUD operations on the `aposAttachments` collection. Note that the reference uses the singular version of the name.
+:::
+
+Next, we are giving our database a MongoDB CRUD method saying that we want to update one document within the database. You can read about other methods within the [MongoDB docs](https://www.mongodb.com/docs/v6.0/crud/). Just make sure that you select your database version from the dropdown.
+
+We are passing two arguments to the MongoDB method. The first is an object that sets the `_id` property to the `_id` of the piece that the user is requesting. Note the use of the underscore `_` - this is because this is a MongoDB, not user, generated id.
```

Comment:
The first is a MongoDB criteria object which ensures our query only matches the document whose `_id` property...

Reply:
^ This edit is still needed

Reply:
Not sure how that happened. Changed.